src/lib/ethereum.ts

```diff
@@ -22,8 +26,11 @@ export const createETHOperationHash = (
 }
 
 export const createTokenOperationHash = (
-  address: string, value: number, contractAddress: string, expireBlock: number, contractNonce: number
+  address: string, value: string, contractAddress: string, expireBlock: number, contractNonce: number
```

Comment:
i to tak po prostu działa że dajemy string zamiast numbera i się ten hex liczy tak samo?

Reply:
No wlasnie bylem pewien ze tak nie bedzie ale sprawdzilem i dziala